pkg/minikube/assets/addons.go

```diff
@@ -47,6 +47,11 @@ func NewAddon(assets []*BinAsset, enabled bool, addonName string) *Addon {
 	return a
 }
 
+// GetAddon get the addon name
+func (a *Addon) GetAddon() string {
```

Comment:
lets change this to Name()
since this does not return the addon but returns a string.

Reply:
Done. Thanks Medya

Reply:
in go style usually there is no need to say Get, and also since the func on an addon, there is no need to say Addon in the name either, so I suggest changing it to just Name()

Reply:
Thanks Medya, I changed it to Name().